src/components/header/modals/ModalConnectWallet.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <astar-simple-modal :show="isModalConnectWallet" title="Select a Wallet" @close="setCloseModal">
+    <div class="wrapper--modal--wallet">
+      <div class="title--account-type">{{ $t('wallet.nativeAccount') }}</div>
+      <div
+        v-for="(wallet, index) in nativeWallets"
```

Comment:
https://user-images.githubusercontent.com/40356749/161081917-c58d03ec-94a8-4b9c-9489-7becabb0a5e1.mov

There is an issue when I try to switch to MetaMask. The wallet should connect naturally instead of showing this behavior.

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/310) PR.
CC: @sirius651 @bobo-k2